packages/universal-cookie-koa/src/index.ts

```diff
@@ -11,7 +11,13 @@ export default function universalCookieMiddleware() {
         if (change.value === undefined) {
           ctx.cookies.set(change.name, null);
         } else {
-          ctx.cookies.set(change.name, change.value, change.options);
+          const koaOpt = (<any>Object).assign({}, change.options);
+          if (koaOpt.maxAge && change.options && change.options.maxAge) {
+            // the standard for maxAge is seconds but koa uses milliseconds
+            koaOpt.maxAge = change.options.maxAge * 1000;
```

Comment:
Is it possible to specify that the `maxAge` is defined as milliseconds on the variable name? Like `maxAgeMilliseconds`, is clearer to understand when reading it for the first time